### PR TITLE
CB-10336: Adding platform and plugin guides and other articles to TOC

### PR DESCRIPTION
**Pull request by rakatyal**

### CODE REVIEW

File: www/_data/toc/de-dev-manual.yml [hunk 1]
```diff
@@ -1,21 +1,37 @@
...
+                -   name: "Blackberry 10"
+                    url: "guide/platforms/blackberry10/home.html"
+                -   name: "iOS"
+                    url: "guide/platforms/ios/index.html"
+                -   name: "Ubuntu"
+                    url: "guide/platforms/ubuntu/index.html"
+                -   name: "Windows"
+                    url: "guide/platforms/win8/index.html"
+                -   name: "WP8"
+                    url: "guide/platforms/wp8/home.html"
```
Comment (dblotsky) [inline www/_data/toc/de-dev-manual.yml]: Is OS X supposed to be in this list, or has that content not made it in yet?

Comment (rakatyal) [inline www/_data/toc/de-dev-manual.yml]: That's still in PR.

**Comment by nikhilkh** [comment]
LGTM!

**Comment by rakatyal** [comment]
Merged [here](https://github.com/apache/cordova-docs/commit/f8e4b8b8ee0c9f0a8afef27a7d9a9c48b9274d36).

**Pull request closed without merging**